.readthedocs.yaml

```diff
@@ -0,0 +1,13 @@
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+version: 2
+
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.11"
+  commands:
+    - mkdir --parents $READTHEDOCS_OUTPUT/html/
```

Comment:
Normally, this would be created by the site generator, if you feed it with the target directory.
```suggestion
```